pythonforandroid/recipe.py

```diff
@@ -478,10 +478,10 @@ def unpack(self, arch):
                 elif isdir(extraction_filename):
                     ensure_dir(directory_name)
                     for entry in listdir(extraction_filename):
-                        if entry not in ('.git',):
-                            shprint(sh.cp, '-Rv',
-                                    join(extraction_filename, entry),
-                                    directory_name)
+                        # if entry not in ('.git',): .git folder is required by `setuptools_scm`
```

Comment:
Let's just change this comment with:
```
# Previously we filtered out the .git folder, but during the build process for some recipes
# (e.g. when version is parsed by `setuptools_scm`) that may be needed.
```

So, someone, looking into this in the future, will get a full explaination of what is happening, and we do not keep commented code around.